style.css

```diff
@@ -0,0 +1,85 @@
+body {
+  background-image: url('images/Header bg.png');
```

Comment:
- [ ] Kindly add background-image to the container class to not produce any problems in your style in the next milestones.